packages/core/src/index.js

```diff
@@ -12,6 +12,7 @@ export { default as RenderTexture } from './textures/RenderTexture';
 export { default as BaseRenderTexture } from './textures/BaseRenderTexture';
 export { default as TextureUvs } from './textures/TextureUvs';
 export { default as WebGLSystem } from './renderers/systems/WebGLSystem';
+export { default as FilterSystem } from './renderers/systems/filter/FilterSystem';
```

Comment:
I think we should export this to systems.FilterSystem (like resources, line 2) instead of exporting directly.